jsonschema_gentypes/configuration.py

```diff
@@ -2,10 +2,12 @@
 Automatically generated file from a JSON schema.
 """
 
-from typing import Any, Literal, TypedDict, Union
+from typing import Any, Callable, Literal, Optional, TypedDict, Union
```

Comment:
This file is generated from `jsonschema_gentypes/schema.json` and describes the schema of the configurations file.

Then if we want to be able to configure it only with python code we should get the function definition out of this file :-)

But in fact, I didn't see well how you plan to use it?
Like in the test?

Reply:
@sbrunner ok! I fixed the two other issues you pointed out.

I can totally get the definition out and instead put it in the `configuration` file.

what's the command to generate the schema.json from the configuration file?

I plan to use this exactly how I use it in the test.

Without the `custom_get_name`, you wind up with the `.title` version like `Subresourceuri` as the name and the test fails. I need the only-first-letter-capitalized version like `SubresourceUri`. 

I also invoke this by importing the `process_config` function from a Python script, not from the cli.

Reply:
@sbrunner want me to pull that definition out of the `configuration` file? any other changes?

Reply:
> what's the command to generate the schema.json from the configuration file?

It's the inverse, this file is generated from this schema https://github.com/sbrunner/jsonschema-gentypes/blob/7c7fe0d6f5539c13334580762bfacef292c6edba/jsonschema_gentypes/schema.json

It's done by pre-commit :-)